src/main/java/com/smartbear/readyapi/client/assertions/DefaultResponseSLAAssertionBuilder.java

```diff
@@ -9,7 +9,7 @@
     private ResponseSLAAssertion responseSLAAssertion = new ResponseSLAAssertion();
 
     public DefaultResponseSLAAssertionBuilder(int maxResponseTime) {
```

Comment:
I think we also need to provide constructor/method overrides that take the corresponding String arguments